tests/ulyanov_et_al_2016/test_modules.py

```diff
@@ -222,16 +222,21 @@ def test_UlyanovEtAl2016ConvSequence(subtests):
 
 
 def test_UlyanovEtAl2016JoinBlock(subtests, input_image):
-    in_channels = extract_num_channels(input_image)
+    input_image1 = input_image
+    input_image2 = torch.cat((input_image, input_image), 1)
+    branch_in_channels = (
+        extract_num_channels(input_image1),
+        extract_num_channels(input_image2),
+    )
     channel_dim = 1
     momentum = 1e-1
     configs = (
-        ((in_channels, in_channels), True, ("block1", "block_2")),
-        ((in_channels, in_channels), True, None),
-        ((in_channels, in_channels), False, ("block1", "block_2")),
-        ((in_channels, in_channels), False, None),
+        (True, ("block1", "block_2")),
+        (True, None),
+        (False, ("block1", "block_2")),
+        (False, None),
     )
-    for branch_in_channels, instance_norm, names in configs:
+    for instance_norm, names in configs:
```

Comment:
`for instance_norm, names in itertools.product((True, False), (("block1", "block2"), None))`: